test/e2e/pkg/manifest.go

```diff
@@ -115,6 +115,15 @@ type ManifestNode struct {
 	// pause:      temporarily pauses (freezes) the node
 	// restart:    restarts the node, shutting it down with SIGTERM
 	Perturb []string `toml:"perturb"`
+
+	// Behaviors sets how a validator behaves during consensus at a
+	// certain height. Multiple behaviors at different heights can be used
+	//
+	// An example of a behavior
+	//    "double-prevote,10"
+	//
+	// For more information, look at the readme in the maverick folden
+	Behaviors string `toml:"behaviors"`
```

Comment:
Yeah I did it this way because it matches the format of the flags which I've already coded. How does this work with toml - would it be something like this?
```
[node.validator02.behaviors]
10 = "double-vote"
15 = "double-vote"
```

Reply:
The nicest way to specify it would be like this:

```
[node.validator02]
behaviors = {10 = "double-vote", 15 = "double-vote"}
```

Reply:
PS: this may require the manifest map to be `map[string]string` since I don't think TOML allows keys to be integers. Then convert those to `map[int64]string` when building the `Testnet` struct.